python/lsst/ts/standardscripts/auxtel/scheduler/enable.py

```diff
@@ -0,0 +1,28 @@
+# This file is part of ts_standardscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+from lsst.ts.standardscripts.scheduler.base_enable import BaseEnable
+
+
+class Enable(BaseEnable):
+    """Enable the ATScheduler."""
+
+    def __init__(self, index: int, descr: str) -> None:
+        super().__init__(index, descr, scheduler_index=2)
```

Comment:
These new scheduler-specific subclasses seem too trivial to bother adding to the library. One argument in support of this is the fact that you don't test the subclasses (and I'm not asking you to do so).

I strongly suggest just creating the subclasses in the scripts themselves.

This would work even better if I updated salobj BaseScript so that amain and make_from_cmd_line accepted an extra ``**kwargs`` argument to pass to the base class constructor. I can add that to salobj 7.1 if you like. Then you would not even need trivial subclasses. But for now...

Reply:
https://jira.lsstcorp.org/browse/DM-34479